R/draw_normal_icc.R

```diff
@@ -217,7 +240,13 @@ draw_normal_icc <- function(mean = 0,
   # And error terms, which are truly individual
   epsilon_ij <- rnorm(length(clusters), 0, sd)
 
-  individual_mean + alpha_individual + epsilon_ij
+  result <- individual_mean + alpha_individual + epsilon_ij
+
+  if(!is.null(total_sd)) {
+    rescale_variable_sd(result, total_sd)
```

Comment:
I think Chad's initial objection was that he preferred a post-hoc scaling because it guaranteed an exact total variance. If we use desired total variance just as a way to calculate the missing parameter of between/within then there would be no scaling. Still could be useful (maybe the user wants total + between instead of between + within or ICC + between or whatever) but it's a different thing, no? Can make changes here depending on how people feel. Do we want to issue this?

Reply:
Let's issue it!